decentraland/unity-explorer
# fix: unequip force render

## What does this PR change?

Fixes #1905 

The issue was provoked when setting the wearable category to `forceRender:true` followed by un-equipping the wearable.
An attempt to fix issue was made here: https://github.com/decentraland/unity-explorer/pull/2363
However it didn't fix it completely since you were able to un-equip the wearable from the grid.
The force render removal is now handled at `BackpackBusController` which captures all un-equip performed events.

## How to test the changes?

You can try this pr using a specific launch parameter to be able to preview a list of wearables in the backpack:
```
--self-preview-wearables urn:decentraland:matic:collections-v2:0x90a7982d7029008c1feaf4e79edc4144ee69556c:0,urn:decentraland:matic:collections-v2:0x14487cadc990b19fe529da04b63d7be091d1c166:1,urn:decentraland:matic:collections-v2:0x84a1d84f183fa0fd9b6b9cb1ed0ff1b7f5409ebb:5,urn:decentraland:matic:collections-v2:0xc11b9d892e12cfaca551551345266d60e9abff6e:1,urn:decentraland:matic:collections-v2:0xc43914be599f3f8fe4956aa3ec9d6a6aead1edfa:2,urn:decentraland:matic:collections-v2:0xded1e53d7a43ac1844b66c0ca0f02627eb42e16d:3,urn:decentraland:matic:collections-v2:0x8169abdb6134748d1a950d530705934ccfd98428:1,urn:decentraland:matic:collections-v2:0xc11b9d892e12cfaca551551345266d60e9abff6e:3,urn:decentraland:matic:collections-v2:0x722dbe4677dd3069c1277946aab57b2e8553bd0f:3
```

1. Launch the explorer
2. Open the backpack
3. Set any category to force render by clicking the eye icon (it should be green)
4. Unequip that wearable
5. Keep equiping other wearables of other categories that might overlap with the previous wearable
6. Check that they dont overlap

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
